src/Raven.Server/Documents/Handlers/Debugging/IdentityDebugHandler.cs

```diff
@@ -40,26 +40,23 @@ public Task GetIdentities()
         }
 
 
-        [RavenAction("/databases/*/identity/next", "GET", AuthorizationStatus.ValidUser, IsDebugInformationEndpoint = true)]
+        [RavenAction("/databases/*/identity/next", "GET", AuthorizationStatus.ValidUser)]
         public async Task NextIdentityFor()
         {
             var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
 
             if (name[name.Length - 1] != '|')
                 name += '|';
             
-            (var clusterEtag, var clusterId) = await Database.ServerStore.GenerateClusterIdentityAsync(name, Database.Name);
+            var (_, _, id) = await Database.ServerStore.GenerateClusterIdentityAsync(name, Database.Name);
 
             using (ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
             using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
             {
                 writer.WriteStartObject();
 
-                writer.WritePropertyName("ClusterEtag");
-                writer.WriteInteger(clusterEtag);
-                writer.WriteComma();
-                writer.WritePropertyName("ClusterId");
-                writer.WriteString(clusterId);
+                writer.WritePropertyName("Index");
```

Comment:
Index? That is the new identity value here, no?
In that case, you want to make sure that this is clearly spelled out.
Because `Index` is usually referred to the _raft_ index.